### ISSUE TITLE
Make the dropdown select tree expanded up to the currently selected item.

### ISSUE THREAD

**Issue by PasinduPrabhashitha** [dowjones/react-dropdown-tree-select#608]
Hey,
Once we select the item from the dropdown, the tree will be collapsed. On the second round when we click on the select component we should again go from the root level to leaf level to check which item has selected. Is there any way to directly expand the dropdown up to the selected level if there any selected item, Once we click the component?

**Comment by mrchief** [comment]
I don't think so. It's more meant for a search and select use case for heavy data nodes, where opening and manually scrolling are tedious. For less intensive data, a little bit of scrolling would be need but then, search and select is is the way to go.
